lib/node_modules/@stdlib/blas/ext/base/cfill/lib/cfill.native.js

```diff
@@ -58,9 +58,10 @@ var addon = require( './../src/addon.node' );
 * var im = imagf( y );
 * // returns 10.0
 */
-function cfill( N, alpha, x, stride ) {
-	var view = reinterpret( x, 0 );
-	addon( N, alpha, view, stride );
+function cfill( N, alpha, x, strideX ) {
+	var view;
+	view = reinterpret( x, 0 );
```

Comment:
```suggestion
	var view = reinterpret( x, 0 );
```

It's fine to declare and assign here.